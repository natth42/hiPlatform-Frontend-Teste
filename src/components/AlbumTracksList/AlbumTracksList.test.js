import React from 'react';
import AlbumsTracksList from './AlbumTracksList';
import { shallow } from 'enzyme';

it('renders AlbumsTracksList without crashing', () => {
    const wrapper = shallow(<AlbumsTracksList />);
    expect(wrapper).toMatchSnapshot();
});




