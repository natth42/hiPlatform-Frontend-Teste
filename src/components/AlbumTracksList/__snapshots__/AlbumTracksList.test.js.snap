// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AlbumsTracksList without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AlbumsTracksList />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <Table
        hover={true}
        responsive={true}
        responsiveTag="div"
        role="table"
        tag="table"
      >
        <thead>
          <tr>
            <th>
              Música
            </th>
            <th>
              Artistas
            </th>
            <th>
              Duração
            </th>
          </tr>
        </thead>
        <tbody />
      </Table>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <thead>
            <tr>
              <th>
                Música
              </th>
              <th>
                Artistas
              </th>
              <th>
                Duração
              </th>
            </tr>
          </thead>,
          <tbody />,
        ],
        "hover": true,
        "responsive": true,
        "responsiveTag": "div",
        "role": "table",
        "tag": "table",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <tr>
              <th>
                Música
              </th>
              <th>
                Artistas
              </th>
              <th>
                Duração
              </th>
            </tr>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <th>
                  Música
                </th>,
                <th>
                  Artistas
                </th>,
                <th>
                  Duração
                </th>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Música",
                },
                "ref": null,
                "rendered": "Música",
                "type": "th",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Artistas",
                },
                "ref": null,
                "rendered": "Artistas",
                "type": "th",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Duração",
                },
                "ref": null,
                "rendered": "Duração",
                "type": "th",
              },
            ],
            "type": "tr",
          },
          "type": "thead",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "tbody",
        },
      ],
      "type": [Function],
    },
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <Table
          hover={true}
          responsive={true}
          responsiveTag="div"
          role="table"
          tag="table"
        >
          <thead>
            <tr>
              <th>
                Música
              </th>
              <th>
                Artistas
              </th>
              <th>
                Duração
              </th>
            </tr>
          </thead>
          <tbody />
        </Table>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <thead>
              <tr>
                <th>
                  Música
                </th>
                <th>
                  Artistas
                </th>
                <th>
                  Duração
                </th>
              </tr>
            </thead>,
            <tbody />,
          ],
          "hover": true,
          "responsive": true,
          "responsiveTag": "div",
          "role": "table",
          "tag": "table",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <tr>
                <th>
                  Música
                </th>
                <th>
                  Artistas
                </th>
                <th>
                  Duração
                </th>
              </tr>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <th>
                    Música
                  </th>,
                  <th>
                    Artistas
                  </th>,
                  <th>
                    Duração
                  </th>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Música",
                  },
                  "ref": null,
                  "rendered": "Música",
                  "type": "th",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Artistas",
                  },
                  "ref": null,
                  "rendered": "Artistas",
                  "type": "th",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Duração",
                  },
                  "ref": null,
                  "rendered": "Duração",
                  "type": "th",
                },
              ],
              "type": "tr",
            },
            "type": "thead",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": "tbody",
          },
        ],
        "type": [Function],
      },
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
